modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc

```diff
@@ -35,41 +37,41 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. From the *Administrator* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
+. In the *Administrator* perspective of the {product-title} web console, click *Observe* and go to the *Metrics* tab.
 
-. To add one or more queries, do any of the following:
+. To add one or more queries, perform any of the following actions:
 +
 |===
 |Option |Description
 
+|Select an existing query.
+|From the *Select query* drop-down list, select an existing query.
+
 |Create a custom query.
 |Add your Prometheus Query Language (PromQL) query to the *Expression* field.
 
 As you type a PromQL expression, autocomplete suggestions appear in a drop-down list. These suggestions include functions, metrics, labels, and time tokens.
 You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression. You can also move your mouse pointer over a suggested item to view a brief description of that item.
 
-|Add multiple queries. |Select *Add query*.
+|Add multiple queries. |Click *Add query*.
 
-|Duplicate an existing query. |Select the Options menu {kebab} next to the query, then choose *Duplicate query*.
+|Duplicate an existing query. |Click the Options menu {kebab} next to the query, then choose *Duplicate query*.
 
-|Disable a query from being run. |Select the Options menu {kebab} next to the query and choose *Disable query*.
+|Disable a query from being run. |Click the Options menu {kebab} next to the query and choose *Disable query*.
 |===
 
-. To run queries that you created, select *Run queries*. The metrics from the queries are visualized on the plot. If a query is invalid, the UI shows an error message.
+. To run queries that you created, click *Run queries*. The metrics from the queries are visualized on the plot. If a query is invalid, the UI shows an error message.
 +
 [NOTE]
 ====
-Queries that operate on large amounts of data might time out or overload the browser when drawing time series graphs. To avoid this, select *Hide graph* and calibrate your query using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
-====
-+
-[NOTE]
-====
-By default, the query table shows an expanded view that lists every metric and its current value. You can select *˅* to minimize the expanded view for a query.
+* Queries that operate on large amounts of data might time out or overload the browser when drawing time series graphs. To avoid this, click *Hide graph* and calibrate your query by using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
```

Comment:
Give the "what" before the "why": "When drawing time series graphs, queries that operate on large amounts of data might time out or overload the browser."